charts/nginx-ingress/values.schema.json

```diff
@@ -139,6 +148,201 @@
+                  "name": "app-protect-bundles",
+                  "emptyDir": {}
+                }
+              ],
+              "title": "Volumes for App Protect WAF v5",
+              "items": {
+                "type": "object",
+                "$ref": "https://raw.githubusercontent.com/nginxinc/kubernetes-json-schema/master/v1.29.0/_definitions.json#/definitions/io.k8s.api.core.v1.Volume"
+              }
+            },
+            "enforcer": {
+              "type": "object",
+              "properties": {
+                "host": {
+                  "type": "string",
+                  "default": "127.0.0.1",
+                  "title": "Port which the App Protect WAF v5 Enforcer process runs on",
+                  "examples": [
+                    50000
+                  ]
+                },
+                "port": {
+                  "type": "integer",
+                  "default": 50000,
+                  "title": "Port which the App Protect WAF v5 Enforcer process runs on",
+                  "examples": [
+                    50000
+                  ]
+                },
+                "image": {
+                  "type": "object",
+                  "default": {},
+                  "title": "The image Schema",
+                  "required": [
+                    "repository"
+                  ],
+                  "properties": {
+                    "repository": {
+                      "type": "string",
+                      "default": "private-registry.nginx.com/nap/waf-enforcer",
+                      "title": "The repository of the App Protect WAF v5 Enforcer image",
+                      "examples": [
+                        "private-registry.nginx.com/nap/waf-enforcer"
+                      ]
+                    },
+                    "tag": {
+                      "type": "string",
+                      "default": "5.2.0",
+                      "title": "The tag of the  App Protect WAF v5 Enforcer image",
+                      "examples": [
+                        "5.2.0"
+                      ]
+                    },
+                    "digest": {
+                      "type": "string",
+                      "default": "",
+                      "title": "The digest of the  App Protect WAF v5 Enforcer image",
+                      "examples": [
+                        "sha256:2710c264e8eaeb663cee63db37b75a1ac1709f63a130fb091c843a6c3a4dc572"
+                      ]
+                    },
+                    "pullPolicy": {
+                      "type": "string",
+                      "default": "IfNotPresent",
+                      "title": "The pullPolicy for the  App Protect WAF v5 Enforcer image",
+                      "allOf": [
+                        {
+                          "$ref": "https://raw.githubusercontent.com/nginxinc/kubernetes-json-schema/master/v1.29.0/_definitions.json#/definitions/io.k8s.api.core.v1.Container/properties/imagePullPolicy"
```

Comment:
did we not update these to 1.30.x

Reply:
@pdabelf5 looks like that wasn't merged back to main, I think we should do that separately from this PR?